modules/src/music.py

```diff
@@ -10,9 +12,19 @@
 def process(input, entities):
     output = {}
     try:
+
+        auth_url = "https://accounts.spotify.com/api/token"
+        headers = {'Authorization': 'Basic ' + config.SPOTIFY_API_KEY}
+        payload = {'grant_type': 'client_credentials'}
+
+        r = requests.post(auth_url, headers=headers, data=payload)
+
+        key = r.json()['access_token']
+
         music = entities['music'][0]['value']
         with requests_cache.enabled('music_cache', backend='sqlite', expire_after=3600):
-            r = requests.get('https://api.spotify.com/v1/search?q=' + music + '&type=track')
+            headers = {'Authorization': 'Bearer ' + key}
+            r = requests.get('https://api.spotify.com/v1/search?q=' + music + '&type=track&limit=1', headers=headers)
```

Comment:
Based on #208 I assumed we only used the first result. But looking at the rest of the module, I guess not.